arduino/arduino/arduino.ino

```diff
@@ -1,16 +1,93 @@
 
 const int tempSensorPin = A0;
+const int pirSensorPin = 3;
+const int micSensorPin = 5;
+const int calibrationTime = 30;
+long interval = 20000;
+long unsigned int pirPause = 5000;
+long unsigned int micPause = 5000;
+
 unsigned long prevTime = 0;
 unsigned long currTime = 0;
-long interval = 20000;
+boolean pirIsTriggered = false;
+boolean savePIRTime = false;
+long unsigned int pirIn;
+boolean micIsTriggered = false;
+boolean saveMICTime = false;
+long unsigned int micIn;
 
 void setup() {
   Serial.begin(115200);
+  pinMode(pirSensorPin, INPUT);
+  pinMode(micSensorPin, INPUT);
+  
+
+  //give the sensor some time to calibrate
+  Serial.print("calibrating sensor ");
+  for(int i = 0; i < calibrationTime; i++){
```

Comment:
No good reason other than to be somewhat careful. I think normal ranges are anywhere between 5 - 60 seconds. No documentation with sensor so I just split the middle.